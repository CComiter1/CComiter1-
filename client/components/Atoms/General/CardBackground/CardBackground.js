import styled from 'styled-components'

export default styled.div`
  background-color: white;
  padding: 1rem .5rem;
  margin: 7px 5px
  border-radius: 15px;
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  width: 100%;
`
