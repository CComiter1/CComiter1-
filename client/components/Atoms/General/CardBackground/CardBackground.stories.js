import React from 'react'
import { storiesOf } from '@storybook/react'

import CardBackground from './CardBackground'

storiesOf('Atoms/General/CardBackground', module)
  .add('Empty CardBackground', () => <CardBackground />)
  .add('CardBackground with Text', () => <CardBackground>Hello World<br /><br />Looking good</CardBackground>)
