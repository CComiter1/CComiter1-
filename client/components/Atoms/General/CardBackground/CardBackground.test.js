import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import CardBackground from './CardBackground'

describe('CardBackground', () => {
  const component = renderer.create(<CardBackground>Press Me</CardBackground>)
  it('renders corrctly', () => {
    expect(component.toJSON()).toMatchSnapshot()
  })
})
