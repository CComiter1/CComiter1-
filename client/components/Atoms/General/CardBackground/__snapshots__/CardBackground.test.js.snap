// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBackground renders corrctly 1`] = `
.c0 {
  background-color: white;
  padding: 1rem 3rem;
  border-radius: 15px;
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  -webkit-transition: 0.3s;
  transition: 0.3s;
}

<div
  className="c0"
>
  Press Me
</div>
`;
